scripts/aws/ebs-warmer.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+# Ref: http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ebs-initialize.html
+
+function log() {
+    echo "[$(date --rfc-3339=seconds)] $1"
+}
+
+log "Warming up RWD EBS volume..."
+
+find /opt/rwd-data/nhd/Main_Watershed/* -type f -print0 \
+    | xargs -0 -L1 -t cat >/dev/null
```

Comment:
I saw an increase in throughput by adding `-P0` to `xargs`. It allows `xargs` to spawn more instances of `cat` at a time.